.github/workflows/swift.yaml

```diff
@@ -24,8 +24,12 @@ jobs:
         - Development-Unit-WorkflowRxSwiftTestingTests
 
     steps:
-    - uses: actions/checkout@v1
-
+    - uses: actions/checkout@v3
+        
+    - uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: '2.7.6' # Not needed with a .ruby-version file
```

Comment:
i don't really understand this comment – what does it mean exactly?

Reply:
If you have a .ruby-version file in the repo it can read that to know what version the action should load.
I was hoping we could get our build to work without having to hard code a ruby version so I didn't introduce that file.